src/hotspot/share/gc/shared/space.hpp

```diff
@@ -548,6 +550,9 @@ class ContiguousSpace: public CompactibleSpace {
   // Size computations: sizes in bytes.
   size_t capacity() const        { return byte_size(bottom(), end()); }
   size_t used() const            { return byte_size(bottom(), top()); }
+  size_t live() const            {
+    return used() - _dead_space;
+  }
```

Comment:
Move it a few lines down, so `capacity`, `used`, `live` line up?

Reply:
Ok. I did my best to make the code look nice there :)